docs/build/start-building/deploy-your-application-to-the-filecoin-testnet.md

```diff
@@ -31,17 +33,47 @@ The high-level steps required to join the testnet are:
 
 Our teammates have also had success running `m5ad.2xlarge` instances on AWS. In order to significantly boost the initial chain sync speed, make sure to keep `~/.lotus` on a fast drive (ideally a SSD -- nvme is best!) and consider temporarily applying the patch listed in [this issue comment](https://github.com/filecoin-project/lotus/issues/3263#issue-684587473).
 
-2. [**Run the lotus daemon.**](https://docs.filecoin.io/how-to/join-a-network/#get-started)
+### (2) Download the latest Filecoin chain state export
+
+Syncing the Filecoin chain from scratch can be a time-consuming process. Currently, chainsync takes about 1/4th of the lifetime of the network to complete. That is, a network 100 days old will take about 25 days to sync from scratch. Instead of syncing from scratch, we recommend that developers looking to deploy their applications quickly download a chain state export generated by the Protocol Labs team and initiate their lotus nodes with this state export. We are working on a more robust solution for mainnet. For now, this is the easiest way to avoid a multi-day setup process to get onto the Filecoin testnet.
+
+Download the latest state snapshot here:
+
+| Snapshot Date | Download Link                                                                                                                                                                                                                        | Size (GiB) | sha256sum                                                        |
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ---------- | ---------------------------------------------------------------- |
+| 2020-09-05    | [https://very-temporary-spacerace-chain-snapshot.s3-us-west-2.amazonaws.com/Spacerace_2020-09-05_10-54-00-UTC.car](https://very-temporary-spacerace-chain-snapshot.s3-us-west-2.amazonaws.com/Spacerace_2020-09-05_10-54-00-UTC.car) | 12.6       | 8815c8b20c854323501fef52885cd2edf139c0a93e26918e2fd9a5478f5bdf24 |
+
+### (3) Run the lotus daemon with the state snapshot
+
+Make sure you have a clean lotus install.
+
+```
+rm -rf ~/.lotus
+```
+
+Then run the lotus daemon passing in the file name for the state snapshot you just downloaded.
+
+```
+lotus daemon --import-snapshot Spacerace_2020-09-05_10-54-00-UTC.car
+```
+
+### (4) Wait for the Filecoin chain to sync
+
+Because you are running the lotus daemon with a chain state snapshot, the chain sync process should be much faster than if you were syncing the chain from scratch. However, you will still need to sync the chain from the point at which the snapshot was generated to the current chain head. [You can run `lotus sync wait` to see the current sync status](https://docs.filecoin.io/how-to/join-a-network/#chain-sync).
```

Comment:
```suggestion
Because you are running the lotus daemon from a known chain-state snapshot, the chain sync process will be significantly faster than if you were syncing the chain from scratch. However, you will still need to sync the chain from the point at which the snapshot was generated to the current chain head. [You can run `lotus sync wait` to see the current sync status](https://docs.filecoin.io/how-to/join-a-network/#chain-sync).
```